### PR TITLE
MissionSystem X Interactables, NursePatrol Logic Update, + Misc. Cleanup

### PR DESCRIPTION
**Pull request by jurcaua**
The scene `MAIN`  is now the main scene we should use I believe, so let's switch to using that for future development and level design!

Main things added:
* Mission system now will spawn the box interactable and the mission will listen to the `OnInteractEnd` event from each to see which one is the last to be collected, spawning the remaining enemies afterwards.
* `NursePatrol.cs` will now not just follow the player, but go to the position they first saw them and only continue chasing if the player is still in their view.
* `MissionCheats.cs` is a fun little thing that adds little menus at the top where we can have shortcuts for different things.
  * image
* In `Constants.cs` we should now be using `public const string` instead of `public static string`. I realized this when trying to use it in `MissionCheats.cs` because only constants can be used in some places vs static strings (which can be changed at any point).

### CODE REVIEW

File: Assets/Scripts/NursePatrol.cs [hunk 1]
```diff
@@ -67,8 +79,6 @@ void GotoNextPoint()
     // Follows Target if they are in the field of vision of the nurse, called in FieldOfVision.cs
```
Comment (chadleytan) [inline Assets/Scripts/NursePatrol.cs]: ChaseTarget is not being called in FieldOfVision anymore right?

Comment (jurcaua) [reply Assets/Scripts/NursePatrol.cs]: Yea instead we call it in `NursePatrol`, and `FieldOfVision` instead calls `HandleTargetsInRange`.

**Review by chadleytan — APPROVED** [review]
Everything looks noiceeee to me other than that one comment!

File: Assets/Scripts/NursePatrol.cs (hunk 1 above)
Comment (chadleytan) [reply Assets/Scripts/NursePatrol.cs]: Ok yeah, the comment just confused me for a sec but I figured it out after!

**Pull request merged**